misc/python/materialize/output_consistency/output/format_constants.py

```diff
@@ -7,9 +7,8 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-OUTPUT_LINE_SEPARATOR_1 = "==============="
-OUTPUT_LINE_SEPARATOR_2 = "*****"
-OUTPUT_LINE_SEPARATOR_3 = "-----"
+OUTPUT_LINE_SEPARATOR_MAJOR = "=============================="
+OUTPUT_LINE_SEPARATOR_MINOR = "------------------------------"
```

Comment:
Does this work with buildkite output? https://buildkite.com/docs/pipelines/managing-log-output

Reply:
Thanks for the hint, will check and adjust.

Reply:
Tried it out, they don't collide but also don't bring benefits. I think the issue is that the string is prefixed with the timestamp:

```
[2023-05-30T10:10:04Z] -- +++ Test summary
[2023-05-30T10:10:04Z] -- 32985/32985 queries passed in mode 'LIVE_DATABASE'.
```

Reply:
Is that the timestamp that buildkite implicitly adds? If so, it shouldn't matter. The `--` looks strange though.

Reply:
You are right. ✅ Adjusted.